packages/spiderable/spiderable_client_tests.js

```diff
@@ -0,0 +1,84 @@
+Tinytest.add("spiderable - default hooks registered", function (test, expect) {
+  test.equal(
+    _.keys(Spiderable._onReadyHook.callbacks).length,
+    2
+  );
+});
+
+Tinytest.add("spiderable - is not ready while initial subscriptions aren't started", function (test, expect) {
+  Spiderable._initialSubscriptionsStarted = false;
```

Comment:
There's nothing that reverts this when this test is done. Tests shouldn't leave side effects in global state.